MPF.Core/InfoTool.cs

```diff
@@ -1382,12 +1382,16 @@ public static bool WriteProtectionData(string? outputDirectory, string? filename
                     path = Path.Combine(outputDirectory, $"!protectionInfo{filenameSuffix}.txt");
 
                 using var sw = new StreamWriter(File.Open(path, FileMode.Create, FileAccess.Write));
-                foreach (var kvp in info.CopyProtection.FullProtections)
+
+                List<string> sortedKeys = [.. info.CopyProtection.FullProtections.Keys.OrderBy(k => k)];
+                foreach (string key in sortedKeys)
                 {
-                    if (kvp.Value == null)
-                        sw.WriteLine($"{kvp.Key}: None");
+                    int pathRoot = (Path.GetPathRoot(key) ?? String.Empty).Length;
```

Comment:
Can you please add a setting for this? This changes the default behavior. I'm okay with keeping the sorting outside of the setting gate.

Reply:
So you want drive letters shown as default?
I will look into adding a checkbox for "Hide drive letter" in the Protection settings in the options window.

Reply:
This will also need a new MPF.Check flag?

Reply:
Optional, but preferred.